governance/reviews/template.md

```diff
@@ -23,171 +11,213 @@ Exemplary: project has an extraordinary level of governance development and impl
+whether the project is using Prow or Sherriff (both of which are good), but if not
+you can't verify ownership. Parts of the governance documentation that talk about
+assigning permissions is also good, but the absence of those isn't notable. --->
 
-Governance Drift can occur when the executed and observable governance of a project deviates from the documented governance of the project.
+### Code of Conduct
 
-The project [does/does] not experience governance drift as indicated by...
+* Document adoption of the CNCF Code of Conduct
+* CNCF Code of Conduct is cross-linked from other governance documents.
 
-#### Ownership
+<!--- The CNCF CoC should be present in all appropriate places in the project, which 
+includes their Community or main repository, and public website at a minimum.  Other
+community and governance documentation should refer to the CoC; for example, role
+removal process should mention that people can be removed for CoC violations.  
+If the project is handling their own CoC reports rather than just sending them 
+directly to the CNCF CoCC, you need to check if this is adequately documented, including
+who receives the reports and how and where they're evaluated. --->
 
-<!--- Request that CNCF staff carry out an audit (via Sheriff) that the explicit governance of the project matches GitHub permissions. Check both that all listed maintainers, owners, and other leaders have the level of ownership or approvership that they are supposed to. Also check that there aren't individuals who have broad permissions that aren't explained by any official project role.  Not applicable for projects joining the CNCF. --->
+### Subprojects
 
-The project's ownership evaluation [did/did not] leverage Sheriff, the CNCF GitHub permission auditing tool.
+* All subprojects, if any, are listed.
+* If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.
 
-The project's permissions and ownership settings and files [are/are not] appropriate for the stated governance. Specifically, ...
+<!--- If the project has subprojects, plugins, or other divisions define them here. For each, is ownership and operation of clearly described? Are any standing committees/teams fully described, including listing their members? Does it conform to, align, and is it within scope of the governance expectations of the project?  Does is list communications channels and standing meetings?
+A subproject audit is often very useful to projects because this part of things tends to 
+bit-rot quite a bit, and you can help them see where they need to update it. 
+If the project has a full listing of suboprojects use the text below.  Completeness hinges on whether subproject listings include full ownership, communications, and alignment info. --->
 
-### Maintainer List(s)
+The project maintains a [listing of subprojects](LINK).  This listing appears to be [Very Complete|Mostly Complete|Incomplete].
```

Comment:
```suggestion
The project maintains a [listing of subprojects - add link].  This listing appears to be [Very Complete|Mostly Complete|Incomplete].
```

Just a nit. With the way it was formatted it looked like it was already a link, not something that needs to be added.

Reply:
I followed the way we handle links to be filled in for Project Templates.